packages/processors/src/helpers/tokenMath.ts

```diff
@@ -32,3 +32,22 @@ export const calculateAmountInUsd = (
 
     return amountInUsd.toString();
 };
+
+/**
+ * Calculates the amount in token
+ * @param amountInUSD - The amount in USD
+ * @param tokenPriceInUsd - The price of the token in USD
+ * @param tokenDecimals - The number of decimals the token has
+ * @returns The amount in token
+ */
+export const calculateAmountInToken = (
+    amountInUSD: string,
+    tokenPriceInUsd: string | number,
```

Comment:
should this always be string ?

Reply:
i left it open because pricing provider returns number because coingecko returns a number (anyways, it doesn't affect the purpose of the function)

Reply:
oh, got it , now i remember